src/index.js

```diff
@@ -3,12 +3,17 @@ import ReactDOM from 'react-dom/client';
 import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
+import {BrowserRouter ,Routes, Route}  from 'react-router-dom'
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>
+  <BrowserRouter>
+    <Routes>
+      <Route path="/" element={<App />} />
+      <Route path="blocks" element={<App />} />
+      <Route path="transactions" element={<App />} />
```

Comment:
I guess the routes are temporary? will we have separate component for each route?

Reply:
Except the first route others are temporary. We will fix them along the way(i.e when we add support for them)